README.md

```diff
@@ -2,3 +2,68 @@
 
 This library a simple interface to interact with DIDs that conform to the DID-provider interface.
 
+## Installation
+
+```sh
+npm install dids
+```
+
+## Example usage
+
+```js
+import { DID } from 'dids'
+import IdentityWallet from 'identity-wallet'
+
+// See https://github.com/3box/identity-wallet-js
+const wallet = new IdentityWallet(async () => true, {})
+const alice = new DID(wallet.getDidProvider())
+
+// Authenticate with the provider
+await alice.authenticate()
+
+// Read the DID string - this will throw an error if the DID instance is not authenticated
+const aliceDID = alice.DID
+
+// Create a JWS - this will throw an error if the DID instance is not authenticated
+const jws = await alice.createJWS({ hello: 'world' })
+```
```

Comment:
Neat! Quite clear example. Maybe we should include something that shows that objects that include CIDs are supported as well?

Reply:
👍 Yes good idea